src/project-management/project-management.ts

```diff
@@ -152,16 +157,41 @@ export class ProjectManagement implements FirebaseServiceInterface {
    * Lists summary of all apps in the project
    */
   public listAppMetadata(): Promise<AppMetadata[]> {
-    throw new FirebaseProjectManagementError(
-      'service-unavailable', 'This service is not available');
+    return this.requestHandler.listAppMetadata(this.resourceName)
+      .then((responseData) => this.transformResponseToAppMetadata(responseData));
   }
 
   /**
    * Update display name of the project
    */
-  public setDisplayName(displayName: string): Promise<void> {
-    throw new FirebaseProjectManagementError(
-      'service-unavailable', 'This service is not available');
+  public setDisplayName(newDisplayName: string): Promise<void> {
+    return this.requestHandler.setDisplayName(this.resourceName, newDisplayName);
+  }
+
+  private transformResponseToAppMetadata(responseData: any): AppMetadata[] {
+    this.assertListAppsResponseData(responseData, 'listAppMetadata()');
+
+    if (!responseData.apps) {
+      return [];
+    }
+
+    return responseData.apps.map((appJson: any) => {
+      assertServerResponse(
+          validator.isNonEmptyString(appJson.appId),
+          responseData,
+          `"apps[].appId" field must be present in the listAppMetadata() response data.`);
+      assertServerResponse(
+          validator.isNonEmptyString(appJson.platform),
+          responseData,
+          `"apps[].platform" field must be present in the listAppMetadata() response data.`);
+      return {
+        appId: appJson.appId,
+        displayName: appJson.displayName,
+        platform: APP_PLATFORM_MAP[appJson.platform] || AppPlatform.PLATFORM_UNKNOWN,
```

Comment:
@nbegley , @hiranya911 Is there a better way to transform the `platform` string returned by the API response to the `AppPlatform` enum?

Reply:
Doesn't `AppPlatform[appJson.platform]` work (assuming enum values match exactly to the values returned by the API)? Might need an any cast somewhere to make the compiler happy.

Reply:
@hiranya911, that's exactly the reason why I have to use this map to avoid complains from ts compiler. Thanks for the suggestion

Reply:
Try `(AppPlatform as any)[appJson.platform] || AppPlatform.PLATFORM_UNDEFINED`

Reply:
Updated